<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>866</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alcohol Storage</string>
  </property>
  <widget class="QWidget" name="CentralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="StatusLable">
      <property name="text">
       <string>Use login action to connect (CTRL + L)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QTabWidget" name="MainTab">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="MainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>Main tool bar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="LoginAction"/>
   <addaction name="LogoutAction"/>
   <addaction name="CheckCountAction"/>
   <addaction name="QueryAction"/>
  </widget>
  <widget class="QStatusBar" name="StatusBar"/>
  <action name="LoginAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/network-wired.png</normaloff>:/new/prefix1/network-wired.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="toolTip">
    <string>Click to login into database.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="CheckCountAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/task-due.png</normaloff>:/new/prefix1/task-due.png</iconset>
   </property>
   <property name="text">
    <string>Check count</string>
   </property>
   <property name="toolTip">
    <string>Click to list all products with low amout.</string>
   </property>
  </action>
  <action name="QueryAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/libreoffice-base.png</normaloff>:/new/prefix1/libreoffice-base.png</iconset>
   </property>
   <property name="text">
    <string>Query</string>
   </property>
   <property name="toolTip">
    <string>Click to open SQL Query mode.</string>
   </property>
  </action>
  <action name="LogoutAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/new/prefix1/system-shutdown.png</normaloff>:/new/prefix1/system-shutdown.png</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="toolTip">
    <string>Click to logout</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>LoginAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CheckCountAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionCheckCount()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>QueryAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onQueryAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LogoutAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onActionLogout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onActionLogin()</slot>
  <slot>onActionCheckCount()</slot>
  <slot>onQueryAction()</slot>
  <slot>onActionLogout()</slot>
 </slots>
</ui>
